src/App.js

```diff
@@ -1,10 +1,84 @@
 import React from 'react';
 import './App.css';
+import Button from "./Button";
```

Comment:
In de uitwerking hebben ze de componenten in een aparte map 'components' gezet. heb ik ook niet gedaan...